modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -136,6 +136,79 @@ NOTE: This step also creates a variant as described in xref:push-notifications.a
 
 . xref:data-sync.adoc#sync-server-binding[Bind the {mobile-client} to the {sync-service} app] if you want to demonstrate {sync-service}.
 
-include::../generic-obtaining-the-mobile-sdk-config-file.adoc[leveloffset=1]
+//[[downloading-the-mobile-services-configuration-file]]
+= Downloading the Mobile Services Configuration File
+
+. Open your {mobile-client} in Mobile Developer Console.
+. Copy the `mobile-services.json` configuration to your clipboard.
+. Save the contents of the clipboard to a new file called `mobile-services.json`.
```

Comment:
This is for the showcase app,  which will have a mobile-service.js file already in git https://github.com/aerogear/ionic-showcase/blob/master/src/mobile-services.js , don't think there is any need to create mobile-service.json.  Replace the  contents of the config object in mobile-services.js with the contents of mobile-service.json. i.e. everything between the {} inclusive . This is really hard to describe.

Reply:
I don't follow @austincunningham - would you mind making this technical change and I'll review it from a docs point of view (if there is a need)?

Reply:
Actually, I think I know what you are asking. Let me give it a go.

Reply:
As the show case is only an ionic app, there is no need for the configs for the other platforms,

Reply:
Given that there is no need for the configs for the other platforms, I've removed them. In addition to this, I've added a note to this section to make it clear to the reader that this section is only for the Ionic platform.

That addresses your feedback. Can you review it again please?